docs/.eleventy/layouts/main.njk

```diff
@@ -5,25 +5,30 @@ description: "Snowpack is the fast, bundle-free build tool for all kinds of fron
 ---
 
 <div class="grid-body-header">
-  <svg height="64px" style="padding-left: 8px;" viewBox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-      <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
-          <g id="mountain-solid" transform="translate(-1.000000, 0.000000)"fill-rule="nonzero">
-              <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
-          </g>
+  <svg height="64px" style="padding-left: 8px;" viewbox="0 0 640 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-1" stroke="none" stroke-width="1" fill="currentColor" fill-rule="evenodd">
+      <g id="mountain-solid" transform="translate(-1.000000, 0.000000)" fill-rule="nonzero">
+        <path d="M635.92,462.7 L347.92,14.7 C342.03,5.54 331.89,0 321,0 C310.11,0 299.97,5.54 294.08,14.7 L6.08,462.7 C-0.250773249,472.547007 -0.699487627,485.064987 4.91,495.34 C10.522069,505.612419 21.2945349,512 33,512 L609,512 C620.71,512 631.48,505.61 637.09,495.33 C642.699457,485.058495 642.250708,472.543372 635.92,462.7 Z M321,91.18 L406.39,224 L321,224 L257,288 L218.94,249.94 L321,91.18 Z" id="Shape"></path>
       </g>
+    </g>
   </svg>
   <h1 class="header-snowpack">Snowpack</h1>
   <p class="header-snowpack-subtitle">
     The faster frontend build tool.
   </p>
+  <p class="header-snowpack-blurb">
+    Snowpack is a modern front-end build tool that replaces traditional bundlers like Webpack or Parcel. It's fast... really fast. Snowpack starts up in ~50 milliseconds and instantly displays code changes in the browser using JavaScript's native module system(<a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import">ESM</a>). When it's time to deploy, Snowpack works with your favorite tools and bundlers to create a fast, production-ready build. 
```

Comment:
Yes! I think this is it! 1-2 smaller comments but overall 100% on board with this structure for the blurb.

- nit: `Snowpack starts up in ~50 milliseconds` vs. `Snowpack starts up in 50 milliseconds or less`?
- `and instantly displays code changes in the browser using JavaScript's native module system` ***VS.*** `and rebuilds files instantly using JavaScript's native module system`
  - The only reason I harp on "displays changes instantly" vs. "rebuilds changes instantly" is that every build tool claims to support instant updates but they always mean "after we do a crap ton of re-building". So, there is something special about the fact that our actual re-builds are instant, that I think conveys a similar message but in a way that no one else case claim. 
  - The fact that this is specifically what ESM gets us also makes me think that this is an important distinction.
- super duper nit, in the above: `using` vs. `thanks to`?